lib/factories/index.js

```diff
@@ -12,11 +12,12 @@ let { PptFactory } = require('./ppt');
 let { RelsFactory } = require('./rels');
 let { Slide } = require('../slide');
 let { Image } = require('../image');
+let { Text } = require('../text');
```

Comment:
`Text` is a existing function within JavaScript so we should pick something else. 

```javascript
Text
ƒ Text() { [native code] }
```

Reply:
OK, will change.

Reply:
Done. Changed to TextBox.